docs/reference/pc_repository.md

```diff
@@ -0,0 +1,756 @@
+		},
+		{
+			"sublime_text": ">=4000",
+			"tags": "st4-"
+		}
+	]
+}
+```
+
+
+### `platforms`
+
+The `platforms` key allows specifying
+what platform(s) the release is valid for.
+Only the latest version for any given platform will be shown to the user.
+It allows specifying a single platform, or a list of platforms.
+Valid platform identifiers include:
+
+- \*
+- windows
+- osx
+- linux
+
+An architecture suffix (`-x32`, `-x64`) is supported
+but no longer relevant in today's world.
+
+Note that this is an optional property:
+if your releases supports all platforms you do not specify `"*"`,
+but instead omit the property completely.
+
+```json
+{
+	"details": "https://github.com/wbond/sublime_alignment",
+	"releases": [
+		{
+			"platforms": ["osx", "linux"],
+			"sublime_text": "*",
+			"tags": "posix-"
+		},
+		{
+			"platforms": "windows",
+			"sublime_text": "*",
+			"tags": "win32-"
+		}
+	]
+}
+```
+
+
+### `base`
+
+If for some reason one of the releases is from a different repository
+than the top-level "details" key,
+a "base" key may be specified in the release.
+This repository will then be used for the matching tags.
+
+```json
+{
+	"details": "https://github.com/wbond/sublime_alignment",
+	"releases": [
+		{
+			"base": "https://github.com/wbond/sublime_alignment",
+			"sublime_text": "<3000",
+			"tags": true
+		},
+		{
+			"sublime_text": ">=3000",
+			"tags": true
+		}
+	]
+}
+```
+
+### `version`, `url`, `date`
+
+For manually created release zips files,
+each individual "version" needs to be specified with
+a "url" to the zip, and a "date".
+Because this requires an update of the repository file for each release
+it is not allowed in the default repository.
+
+Versions needs to be numbered in accordance with [semver](https://semver.org). 
+
+The URL needs to be a zip file containing the package.
+It is permissible for the zip file to contain a single root folder
+with any name.
+All files will be extracted out of this single root folder.
+This allows zip files from GitHub, GitLab and Bitbucket to be used a sources.
+The URL can be a relative path
+from the location of the repository.json file that specifies it.
+
+The date must be in the form "YYYY-MM-DD HH:MM:SS" and should be UTC.
+
+```json
+{
+	"packages": [
+		{
+			"details": "https://github.com/wbond/sublime_alignment",
+			"releases": [
+				{
+					"version": "2.0.0",
+					"url": "https://codeload.github.com/wbond/sublime_alignment/zip/v2.0.0",
+					"date": "2011-09-18 20:12:41",
+					"sublime_text": "*"
+				}
+			]
+		},
+		{
+			"details": "../alignment",
+			"releases": [
+				{
+					"version": "2.0.0",
+					"url": "./downloads/alignment-2.0.0.zip",
+					"date": "2011-09-18 20:12:41",
+					"sublime_text": "*"
+				}
+			]
+		}
+	]
+}
+```
+
+### `asset`
+
+If your package requires a build or compile step,
+you might want to provide your releases via pre-compiled zip archives
+or sublime-package files.
+To do so, specify an `asset` key describing
```

Comment:
Some of the JSON keys in this page use code spans, where others are double-quoted. Pick one or the other.

Reply:
I think I have this covered now 🤞🏻